rust/trezor-client/.gitignore

```diff
@@ -0,0 +1,2 @@
+target/
+Cargo.lock
```

Comment:
i would think we don't want to exclude `Cargo.lock`?

sure, the situation is different when we won't be distributing binaries, but still seems that it might help, e.g., in CI

Reply:
Added `Cargo.lock`